### PR TITLE
Network change - overlay

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow.js [hunk 1]
```diff
@@ -189,7 +192,7 @@ class MetamaskFlow extends Component {
             <span className={classes.channelSelectionTitle}>Recommended</span>
             <ChannelSelectionBox
               title="Channel Balance"
-              description={`You have ${this.parseChannelBalFromPaymentCard()} AGI in you channel. This can be used for running demos across all the services from this vendor.`}
+              description={`You have 1 AGI in you channel. This can be used for running demos across all the services from this vendor.`}
```
Comment (chetankothari) [inline src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow.js:195]: Do you want to dynamically show this?

File: src/components/common/NetworkChangeOverlay/index.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  showMetaMaskConfigMismatchOverlay = () => {
+    const { wallet } = this.props;
+    if (wallet && wallet.type !== walletTypes.METAMASK) {
+      return false;
+    }
+
+    const web3Provider = window.ethereum;
+    if (!web3Provider) {
+      return false;
```
Comment (chetankothari) [inline src/components/common/NetworkChangeOverlay/index.js]: You need to return consistent structure data structure for a function.

File: src/config/Network.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const networks = {
+  ROPSTEN: "ROPSTEN",
```
Comment (chetankothari) [inline src/config/Network.js]: Do you need these constants ?

**Review by chetankothari — APPROVED** [review]
Minor suggestions. Once you address them it should be good to merge.

File: src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow.js (hunk 1 above)
Comment (Vivek205) [reply src/components/ServiceDetails/AboutService/ServiceDemo/Purchase/ExpiredSession/MetamaskFlow.js:195]: This is handled in https://github.com/singnet/refactored-dapp/pull/189

File: src/config/Network.js (hunk 3 above)
Comment (Vivek205) [reply src/config/Network.js]: https://github.com/singnet/refactored-dapp/pull/184/commits/e82e1f8abc060c8f9402b50f18a23d34e0386ee3

File: src/components/common/NetworkChangeOverlay/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/common/NetworkChangeOverlay/index.js]: https://github.com/singnet/refactored-dapp/pull/184/commits/a9259c45e3e1e01c6d7b9d50299f88607e632546

**Pull request merged**